plugin/repeater/repeater.talon

```diff
@@ -3,3 +3,6 @@
 <number_small> times: core.repeat_command(number_small - 1)
 (repeat that | twice): core.repeat_command(1)
 repeat that <number_small> [times]: core.repeat_command(number_small)
+
+again: core.repeat_phrase()
+^repeat phrase <number_small> times$: core.repeat_phrase(number_small - 1)
```

Comment:
Do we need an anchor here?

Reply:
You'd have to ask Ryan 😄

I presume it might behave in a confusing manner if it wasn't anchored. I actually haven't tested what would happen if you used this in the middle of a phrase.

I suppose the other one should be anchored too if this one is.

Reply:
I don't think the `- 1` should be here; `repeat phrase 1 time` shouldn't be a noop. The minus one is for ordinals.